Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM elixir:1.3.4
+RUN mkdir -p /app
+WORKDIR /app
+CMD ["iex", "-S", "mix"]
+COPY . /app
+RUN mix local.hex --force && \
+    mix deps.get && \
+    mix compile && \
+    mix credo --strict
```

Comment:
How so? Do you have an example of bad code? Is it bad because it's hard to read or leads to bugs?